projects/_posts/2014-03-21-Example-Project.md

```diff
@@ -9,6 +9,7 @@ author: [ "@wasabifan" ] # your GitHub user name
 programming_language: "JavaScript" # The programming language used in this project
 
 youtube_video_id: rrNaLfE9PWA # The video ID of the YouTube video to be displayed with this post
+# thumbnail_override: "/images/logo_ev3dev_mono.png" # If you don't have a YouTube video (or the video thumbnail isn't good) you can uncomment this line to set your own image for the project.
```

Comment:
We have a projects folder under `/images/`, so let make the example

    /images/my-project/my-image.png